### DISCUSSION TITLE
Turns out you cannot get the badge if you marked your own comment as an answer

### DISCUSSION THREAD

**Discussion by number0x0001** [Schweinepriester/github-profile-achievements#38] [General]
This looks like a bug. If you mark your own comments as answers, and then others mark yours as answers on other questions, you will not get the badge for some reason. You have to go on a roundabout loop to unmark your answers and then you can have a shot at getting the badge.

**Comment by lincolnvc** [comment]
I didn't identify this type of problem, from what I understand, just comment on the discussion of others and you already win the seal.

**Reply by number0x0001** [reply]
If you've marked your own as answers in the past, you will not be able to get the badge if others mark your comments as answers later on.

**Reply by lincolnvc** [reply]
Got it, you really have this problem
